build.gradle

```diff
@@ -223,7 +223,7 @@ def versions = [
   springfoxSwagger: '3.0.0',
   testcontainers  : '1.15.2',
   jetty           : '9.4.53.v20231009',
-  netty           : '4.1.104.Final'
+  netty           : '5.0.0.Alpha2'
```

Comment:
not really keen on an alpha version going though, recommend waiting a month or two until it hits stable. I imagine all the dependancies will get a 5.0.0 by then too.